lib/ssl/src/ssl.erl

```diff
@@ -42,9 +42,8 @@ Example:
 
 See [Using SSL](using_ssl.md) for detailed usage and more examples of this API.
 
-Special Erlang node configuration for the application can be found in [ssl application reference](ssl_app.md).
-
-
+Special Erlang node configuration for the application can be found in
```

Comment:
I realized while reviewing that the sentence above  saying "See [Using SSL] "  maybe should say [User Guide Examples] instead. As the old user guide chapter's have been converted into several user guides and this one is called Examples, although the file still has the old name.

Reply:
Changed.